requirements/base.in

```diff
@@ -24,3 +24,4 @@ python-magic
 rules
 sentry-sdk[django]
 shapely
+soil_id @ https://github.com/techmatters/soil-id-algorithm/archive/ddf718a.zip
```

Comment:
suggestion: it seems from the rest of the file that it is idiomatic in the python ecosystem to use kebab case rather than snake case, and since the repo itself is named `soil-id-algorithm`, could we also name the package `soil-id-algorithm`?

Reply:
Changed this to `soil-id`.